examples/asset_licences.md

```diff
@@ -0,0 +1,9 @@
+The "Tuffy" font is public domain, was made by Thatcher Ulrich, and was accessed from here: http://www.publicdomainfiles.com/show_file.php?id=13486218041168
```

Comment:
Can we use Tuffy V1.1, which is only about 18KiB in size? Also whenever possible, I'd try to link to the original author: http://tulrich.com/fonts/

Reply:
Absolutely. I will switch it over to that. And yes, I will change that.